src/main/java/org/teamone/onlinestorebuyreadreview/database/mapper/deliveryItem/DeliveryItemRowMapper.java

```diff
@@ -0,0 +1,27 @@
+package org.teamone.onlinestorebuyreadreview.database.mapper.deliveryItem;
+
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.stereotype.Component;
+import org.teamone.onlinestorebuyreadreview.database.entity.Author;
```

Comment:
Delete unused import and set in Intelij an option "Optimize import on Fly"

Reply:
yes, of course